CMakeLists.txt

```diff
@@ -244,49 +244,77 @@ if (NETWORKIT_CLANG_TIDY)
 endif()
 
 if(NETWORKIT_PYTHON)
-	if(NOT EXISTS "${PROJECT_SOURCE_DIR}/networkit/_NetworKit.cpp")
-		message(FATAL_ERROR "networkit/_NetworKit.cpp is missing. Invoke Cython manually.")
-	endif()
+	 foreach (EXT base helpers structures matching graph traversal graphtools
+			 distance dynamics generators centrality community components graphio randomization
+			 linkprediction sparsification viz flow simulation coarsening scd correlation clique
+			 globals independentset engineering stats profiling/stat)
```

Comment:
For better readability, please sort the modules in alphabetical order.

Reply:
I have done so as well as in `setup.py`.